src/Interactivity/directives/class-woo-directive-store.php

```diff
@@ -24,8 +20,11 @@ static function render() {
 			return;
 		}
 
-		$id    = 'store';
-		$store = self::serialize();
-		echo "<script id=\"$id\" type=\"application/json\">$store</script>";
+		$id = 'store';
+		echo sprintf(
+			'<script id="%s" type="application/json">%s</script>',
+			esc_attr( $id ),
```

Comment:
Nitpick, since `$id` is hard-wired to a perfectly innocuous string (`'store'`), the escaping isn't really doing anything (so we're really just doing it to placate the linting and code analysis tool gods). We might consider inlining the whole thing instead:

```suggestion
		echo sprintf(
			'<script id="store" type="application/json">%s</script>',
```

Reply:
I agree, inlining the whole attribute and forgoing the variable makes more sense in this case.

Reply:
Updated in 0d4ace24e.